playground/proxy-bypass/vite.config.js

```diff
@@ -10,6 +12,22 @@ export default defineConfig({
           return false
         },
       },
+      '/asyncResponse': {
+        bypass: async (_, res) => {
+          await timeout(4)
+          res.writeHead(200, {
+            'Content-Type': 'text/plain',
+          })
+          res.end('Hello after 4 ms (async timeout)')
+          return '/asyncResponse'
```

Comment:
I think it's odd to have to return the path of the proxy as the return value here. I think either returning `true` or void should work.

But otherwise this completely handles the cases I care about - thank you!

Reply:
@sapphi-red - please let us know whether you still think returning the path that was just handled makes sense.

Reply:
I think it's fine.